src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -11,13 +16,15 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
-
 	res.render("github-create-branch.hbs", {
 		csrfToken: req.csrfToken(),
 		jiraHost,
 		nonce: res.locals.nonce,
-		title: "Create a Branch"
+		title: "Create a Branch",
```

Comment:
Do we need to pass the title here?

Reply:
Its not necessary I guess, was using it before for the dummy page. I've removed this